app/views/devise/sessions/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2>Log in</h2>
+<h2>Log In</h2>
```

Comment:
![I see what you did there](http://mrwgifs.com/wp-content/uploads/2015/02/I-See-What-You-Did-There-Doctor-Who.gif)